cve_bin_tool/checkers/jre.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for Oracle JRE
+
+References:
+https://www.cvedetails.com/product/19117/Oracle-JRE.html?vendor_id=93
+"""
+from cve_bin_tool.checkers import Checker
+
+
+class JreChecker(Checker):
+    CONTAINS_PATTERNS = []
+    FILENAME_PATTERNS = [r"java", r"java.debuginfo"]
+    VERSION_PATTERNS = [r"openjdk ([0-9]+(.[0-9]+)?(.[0-9]+)?)"]
+    VENDOR_PRODUCT = [("oracle", "java")]
```

Comment:
So, do you want this to be `java` or do you want it to be `jre`? Oracle has listings for both java and jre, but what are you actually detecting, and which CVEs apply to it?

Reply:
Hi @terriko thanks for your review
In this error [ci](https://github.com/intel/cve-bin-tool/pull/1096/checks?check_run_id=2133790185#step:9:7509)
its the error due to inappropriate signatures, and also to download the `rpm files` of the `jre` we need the credentials. So I feel that the `rpm files ` are not getting loaded in the first place itself